app/presenters/step_by_step_page_presenter.rb

```diff
@@ -58,7 +58,7 @@ def summary_list_params
   end
 
   def steps_title
-    return { title: "Steps" } unless step_by_step_page.can_be_edited?
+    return { title: "Steps" } unless step_by_step_page.can_be_edited? && step_by_step_page.steps.present?
```

Comment:
Hasn't it been decided to always show the "Steps" title, even if there aren't any steps?

Reply:
Yes - if you look further down, we return Steps with Reorder link if this conditional fails.

Reply:
Have now refactored it to work the same way as other methods in this presenter (ie a hash which we `merge!` edit links onto if needed)